api/src/tests/event_v2_translation_test.rs

```diff
@@ -262,12 +276,22 @@ async fn test_event_v2_translation_coin_withdraw_event() {
 }
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+#[ignore]
 async fn test_event_v2_translation_account_coin_register_event() {
```

Comment:
I guess we never enabled the translation anyways

Reply:
yes. I am preparing a mainnet proposal.